config/gcp.spc

```diff
@@ -7,7 +7,7 @@ connection "gcp" {
   #   - The standard gcloud SDK `CLOUDSDK_CORE_PROJECT` environment variable, if set; otherwise
   #   - The `GCP_PROJECT` environment variable, if set (this is deprecated); otherwise
   #   - The current active project project, as returned by the `gcloud config get-value project` command
-  #project = "YOUR_PROJECT_NAME"
+  #project = "YOUR_PROJECT_ID"
```

Comment:
@ashishsecdev Can you please also update the same example in `docs/index.md` under the `Configuration` section?

Reply:
> @ashishsecdev Can you please also update the same example in `docs/index.md` under the `Configuration` section?

Sure, let me do that as well.

Reply:
Updated the `example` in docs/index.md under the 'Configuration' section.